apps/wizard/app_pages/chart_diff/show.py

```diff
@@ -160,6 +210,97 @@ def _show_chart_diff_controls(self):
                 else:
                     st.caption(f"**{OWID_ENV.wizard_url}?{query_params}**")
 
+    def _show_metadata_diff(self) -> None:
+        """Show metadata diff (if applicable).
+
+        Come chart-diffs might be triggered by changes in metadata. This allows the user to explore this changes.
+
+        Note that to access the metadata, one needs to retrieve the JSON metadata files from the S3 bucket.
+        """
+        if st.button(
+            "🔎 Metadata differences",
+            f"btn-meta-diff-{self.diff.chart_id}",
+        ):
+            self._show_metadata_diff_modal()
+
+    @st.experimental_dialog("Metadata differences", width="large")  # type: ignore
+    def _show_metadata_diff_modal(self) -> None:
```

Comment:
This looks really nice! Just one nit - I think we could ignore all `id` fields and `updatedAt` when comparing metadata (they're not going to be part of metadata checksum anyway).

Reply:
That makes sense; I'll create the list of columns to ignore somewhere, and use it here and in the checksum estimation. (after you merge https://github.com/owid/etl/pull/2797)

Reply:
Nah, actually I'll abstract your function logic of popping irrelevant fields and will use it here and there